clients/apps/web/src/components/Dashboard/BadgeMessageForm.tsx

```diff
@@ -12,13 +12,9 @@ const BadgeMessageForm = (props: {
   issue: IssueDashboardRead
   onBadgeWithComment: (comment: string) => Promise<void>
 }) => {
-  const [message, setMessage] = useState(
-    `## Funding
```

Comment:
Just a reminder: You'll also have to change this here: https://github.com/polarsource/polar/blob/add-label-from-dashboard/server/polar/integrations/github/badge.py#L115